src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -153,7 +153,7 @@ public function findByIds(array $ids): array
             ->addSelect('mainContact')
             ->addSelect('categories')
             ->addSelect('categoryTranslations')
-            ->addSelect('partial tags.{id, name}')
```

Comment:
we should not built ontop of such features, but I understand that we did try to avoid fetch changer, creator, ... on all entities